README.md

```diff
@@ -31,6 +31,65 @@ For an upgrade the existing file should be overwritten. On major or structural c
 **Note:**
 [v0.32-alpha](https://github.com/nibi79/synologysurveillancestation/releases/tag/v0.32-alpha) is the last release supporting openHAB prior to 2.4.0. If you want to use newer release please consider to upgrade your openHAB.
 
+## Building the plugin
```

Comment:
Would you consider move this entire block to the bottom (prior to support)? I assume, most users would still just download .jar and don't need this explanation... in any case please update table of contents

Reply:
I did move the entire block to the bottom and also updated the TOC.

Reply:
Thanks!  What about building, which is still above? It is also a little sparse... won't I still need Maven, Git and the entire OH core for that?

Reply:
> Thanks! What about building, which is still above? It is also a little sparse... won't I still need Maven, Git and the entire OH core for that?

You are talking about the _developing_ section? Yes that is probably right - but Eclipse should bring most of that. You really want to have a guide on how to setup a proper java environment in the readme? I think that can be easily found out using other channels...

Reply:
I mean moving "Building plugin" and "Building using Docker"down, prior to "Developing" session.

My consideration was: who would ever want to **build** this plugin if not for **developing** it? I assume, most users are happy with JAR download. And if it's true, why place the "Build" section in the first part of the readme (even before the common information such as channel description) and "Developing" in the last part? Why not both together below?

Ultimately, it's your decision. Please don't take this as nitpicking, but a single opinion. 
You've done great job so far and it's not my intention to talk it bad.

Reply:
Makes a lot of sense. Did move the sections to the later part of the documentation